var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -47,7 +59,6 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     conflicts(
         "+python", when="+ipopt+rocm", msg="Python bindings require -fPIC with Ipopt for rocm."
     )
-    variant("logging", default=False, description="Enable/Disable spdlog based logging")
```

Comment:
While this duplicate has been removed, still need to remove duplicate CMake variable in that function.

Reply:
Actually, @haampie just pointed out this morning to me that it is not a good idea to use tag as the concretizer will not treat it the same way as a commit, because tags can be moved around.

Reply:
Nevermind, I see you have added tag *and* commit! I actually didn't even realize this could be done. Sorry!

Reply:
We are all learning! I'm just glad you caught the stray hash